plugins/api/src/get_file_descriptor_info.cpp

```diff
@@ -114,44 +114,41 @@ namespace
             return nullptr;
         }
 
-        return {
-            {"object_path", _p->objPath},
-            {"resource_name", _p->rescName},
-            {"resource_hierarchy", _p->rescHier},
-            {"data_type", _p->dataType},
-            {"data_size", _p->dataSize},
-            {"checksum", _p->chksum},
-            {"version", _p->version},
-            {"file_path", _p->filePath},
-            {"data_owner_name", _p->dataOwnerName},
-            {"data_owner_zone", _p->dataOwnerZone},
-            {"replica_number", _p->replNum},
-            {"is_replica_current", (GOOD_REPLICA == _p->replStatus)},
-            {"replica_status", _p->replStatus},
-            {"status_string", _p->statusString},
-            {"data_id", _p->dataId},
-            {"collection_id", _p->collId},
-            {"data_map_id", _p->dataMapId},
-            {"flags", _p->flags},
-            {"data_comments", _p->dataComments},
-            {"data_mode", _p->dataMode},
-            {"data_expiry", _p->dataExpiry},
-            {"data_create", _p->dataCreate},
-            {"data_modify", _p->dataModify},
-            {"data_access", _p->dataAccess},
-            {"data_access_index", _p->dataAccessInx},
-            {"write_flag", _p->writeFlag},
-            {"destination_resource_name", _p->destRescName},
-            {"backup_resource_name", _p->backupRescName},
-            {"sub_path", _p->subPath},
-            {"special_collection", to_json(_p->specColl)},
-            {"registering_user_id", _p->regUid},
-            {"other_flags", _p->otherFlags},
-            {"condition_input", to_json(_p->condInput)},
-            {"in_pdmo", _p->in_pdmo},
-            {"next", to_json(_p->next)},
-            {"resource_id", _p->rescId}
-        };
+        return {{"object_path", _p->objPath},
+                {"resource_name", _p->rescName},
+                {"resource_hierarchy", _p->rescHier},
+                {"data_type", _p->dataType},
+                {"data_size", _p->dataSize},
+                {"checksum", _p->chksum},
```

Comment:
Let's keep the original formatting for this and wrap it in `clang-format off/on`.

Reply:
Done.